### DISCUSSION TITLE
Dynamic Tags for Button Widget Link Attributes

### DISCUSSION THREAD

**Discussion by lunik2** [elementor/elementor#23595] [Feature Request]
### Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

1. Being able to use the ACF Link field and pass what's inside that array: title, url, target
2. -> To the Button Widget

### Describe the solution you'd like

1. Being able to select multiple values from an array in an ACF link field. (optional, plugins like dynamic.ooo solved that) 
2 Adding a Dynamic Tag to link attributes

### Describe alternatives you've considered

It can be solved using dynamic.ooo, which can extract the state of the target checkbox in ACF outputting "_blank" and passing it on to a classname in the button widget, then using js you can modify the href to add a target _blank attribute if the classname is present.

### Additional context

<img width="287" alt="Bildschirmfoto 2023-08-29 um 08 20 33" src="https://github.com/elementor/elementor/assets/34686543/ae56825f-1499-4a15-bbd8-a71aa7415abf">
<img width="561" alt="Bildschirmfoto 2023-08-29 um 08 21 56" src="https://github.com/elementor/elementor/assets/34686543/c1f7f1eb-eb46-41e0-b65c-dbb6881b6e4b">
<img width="952" alt="Bildschirmfoto 2023-08-29 um 08 24 07" src="https://github.com/elementor/elementor/assets/34686543/3c18374b-67ee-42fd-aa2b-17e15e2f0407">

**Comment by portrayaloflife** [comment]
+1
